Framework/PythonInterface/plugins/algorithms/Abins.py

```diff
@@ -19,42 +18,41 @@
 
 
 # noinspection PyPep8Naming,PyMethodMayBeStatic
-class Abins(PythonAlgorithm, AbinsAlgorithm):
-    _ab_initio_program = None
-    _vibrational_or_phonon_data_file = None
-    _experimental_file = None
-    _temperature = None
-    _bin_width = None
-    _scale = None
-    _sample_form = None
-    _instrument_name = None
-    _setting = None
-    _atoms = None
-    _sum_contributions = None
-    _save_ascii = None
-    _scale_by_cross_section = None
-    _calc_partial = None
-    _out_ws_name = None
-    _num_quantum_order_events = None
-    _max_event_order = None
-    _energy_units = None
-    _autoconvolution = None
+class Abins(AbinsAlgorithm, PythonAlgorithm):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self._sample_form = None
+
+        self._experimental_file = None
+        self._scale = None
+        self._setting = None
 
     def category(self) -> str:
         return "Simulation"
 
     def summary(self) -> str:
         return "Calculates inelastic neutron scattering against 1-D ω axis."
 
+    def seeAlso(self):
+        return ["Abins2D"]
+
     def PyInit(self) -> None:
-        from abins.constants import ONE_DIMENSIONAL_INSTRUMENTS
+        from abins.constants import ALL_SAMPLE_FORMS, ONE_DIMENSIONAL_INSTRUMENTS
         # Declare properties for all Abins Algorithms
         self.declare_common_properties()
 
-        # Declare properties specific to 1D
-        self.declareProperty(name="Autoconvolution", defaultValue=False,
-                             doc="Estimate higher quantum orders by convolution with fundamental spectrum.")
+        # Soon-to-be-deprecated properties (i.e. already removed from 2D)
+        self.declareProperty(name="BinWidthInWavenumber", defaultValue=1.0,
```

Comment:
No, currently this will be called every time the Algorithm is run. Deprecation warnings are usually used to recommend users adopt the future behaviour, which won't be possible yet.